source/retryable-writes/tests/unsupported-multi-statement.yml

```diff
@@ -0,0 +1,87 @@
+data:
+    - { _id: 1, x: 11 }
+    - { _id: 2, x: 22 }
+
+minServerVersion: '3.6'
+
+# Unordered, multi-statement write operations are not supported in MongoDB 3.6
```

Comment:
Clarify that they are not supported within a retryable write session, not in general.